src/controllers/playback/video/index.js

```diff
@@ -326,7 +326,93 @@ export default function (view) {
+    /*const _focus = function () {
+        let lastFocusElement;
+        let timerId;
+
+        function invokeFunc() {
+            // If no focus element is provided, try to keep current focus if it's valid,
+            // otherwise default to pause button
+            const currentFocus = lastFocusElement || document.activeElement;
+            if (!currentFocus || !focusManager.isCurrentlyFocusable(currentFocus)) {
+                lastFocusElement = osdBottomElement.querySelector('.btnPause');
+            }
+
+            if (lastFocusElement) focusManager.focus(lastFocusElement);
+
+            lastFocusElement = undefined;
+            timerId = undefined;
+        }
+
+        const debounced = function (focusElement) {
+            if (focusElement) lastFocusElement = focusElement;
+
+            if (timerId === undefined) {
+                timerId = setTimeout(invokeFunc, 50);
+            }
+        };
+
+        function cancel() {
+            if (timerId !== undefined) {
+                clearTimeout(timerId);
+            }
+            lastFocusElement = undefined;
+            timerId = undefined;
+        }
+
+        debounced.cancel = cancel;
+
+        return debounced;
+    }();*/
+    const _focus = function () {
+        let lastFocusElement;
+
+        const debouncedInternal = debounce(() => {
+            // If no focus element is provided, try to keep current focus if it's valid,
+            // otherwise default to pause button
+            const currentFocus = lastFocusElement || document.activeElement;
+            if (!currentFocus || !focusManager.isCurrentlyFocusable(currentFocus)) {
+                lastFocusElement = osdBottomElement.querySelector('.btnPause');
+            }
+
+            if (lastFocusElement) focusManager.focus(lastFocusElement);
+
+            lastFocusElement = undefined;
+        }, 50);
+
+        const debounced = function (focusElement) {
+            if (focusElement) lastFocusElement = focusElement;
+
+            debouncedInternal();
+        };
+
+        function cancel() {
+            debouncedInternal.cancel();
+            lastFocusElement = undefined;
+        }
+
+        function flush() {
+            const result = debouncedInternal.flush();
+            lastFocusElement = undefined;
+            return result;
+        }
+
+        debounced.cancel = cancel;
+        debounced.flush = flush;
+
+        return debounced;
+    }();
+
+    const _focus2 = function (focusElement) {
+        // If no focus element is provided, try to keep current focus if it's valid,
+        // otherwise default to pause button
+        const currentFocus = focusElement || document.activeElement;
+        if (!currentFocus || !focusManager.isCurrentlyFocusable(currentFocus)) {
+            focusElement = osdBottomElement.querySelector('.btnPause');
+        }
+
+        if (focusElement) focusManager.focus(focusElement);
+    };
```

Comment:
Hmm, do we need to debounce/timeout focus at all? `_focus2` seems to work in webOS 1.2, webOS 5, Firefox 134, Chrome 132.

Maybe timeout was for Internet Explorer? :thinking:

Reply:
Are these changes to the focus function necessary? 

Does wrapping the skip button fix this issue by itself?

Reply:
Technically, there's 2 fixes:
1. Making the Skip button "focusable".
2. Fixing focus on an element passed in `showOsd`.
There're 2 versions: delayed and non-delayed focus. I'm leaning towards the latter, which is currently in use.